linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -554,49 +554,18 @@ private[emitter] object CoreJSLib {
         })
       }
 
-      // getClass()java.lang.Class
-      locally {
-        defineDispatcher(getClassMethodName, Nil, {
-          Return(genCallHelper("objectGetClass", instance))
-        })
-      }
-
-      // clone()java.lang.Object
-      locally {
-        defineDispatcher(cloneMethodName, Nil, {
-          If(genIsScalaJSObjectOrNull(instance), {
-            Return(Apply(instance DOT genName(cloneMethodName), Nil))
-          }, {
-            Throw(genScalaClassNew(CloneNotSupportedExceptionClass,
-                NoArgConstructorName))
-          })
-        })
-      }
-
-      // notify()V and notifyAll()V (final and no-op in Object)
-      locally {
-        for (name <- List(notifyMethodName, notifyAllMethodName)) {
-          defineDispatcher(name, Nil, {
-            If(instance === Null(), Apply(instance DOT genName(name), Nil), Skip())
-          })
-        }
-      }
-
-      // finalize()V
-      locally {
-        defineDispatcher(finalizeMethodName, Nil, {
-          If(genIsScalaJSObjectOrNull(instance),
-              Apply(instance DOT genName(finalizeMethodName), Nil),
-              Skip())
-        })
-      }
-
-      def defineStandardDispatcher(methodName: MethodName, args: List[VarRef],
-          implementationInObject: Option[Tree],
-          maybeImplementingHijackedClasses: List[ClassName]): Unit = {
+      /* A standard dispatcher performs a type test on the instance and then
+       * calls the relevant implementation which is either of:
+       *
+       * - A normal method call if the instance is a normal scala class.
+       * - A default method the relevant hijacked class.
```

Comment:
```suggestion
       * - A method in the relevant hijacked class.
```
Methods in hijacked classes are not *default* methods. They're just ... methods in hijacked classes.

Reply:
Then why are they on the "f" env field?